### ISSUE TITLE
Improve intellisense with grouping by method name

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#71806]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Improve-intellisense-with-grouping-overl/10555208)._

---
Hello,  
I recently tried the CodeRush addon and has grouped methods by method name and sorted on arguments.  
I think that the implementation could be a speed up the display and do it more sense.  
[showcase](https://docs.devexpress.com/CodeRushForRoslyn/images/ir_overloads122725.png)

This option should be voluntary, so that anyone can turn it on/off according to their needs.

Would be great if you could add it and optimize it for everyday use.

---
#### Feedback Bot on 1/8/2024, 06:12 PM: 

(private comment, text removed)

**Comment by Cosifne** [comment]
![image](https://github.com/dotnet/roslyn/assets/24360909/217bdf95-1caa-459f-8d96-4fd6fee021c6)

Looks like a completion feature request.

**Comment by CyrusNajmabadi** [comment]
I don't think we need this.  We already group by method name.  Then, once you select it and hit open-paren, then we have signature help for all the items.

If a user wants to see all the sig help itms at once, that would be an option that needs to go to the hosst (like VS), not us.  We just supply the data, but it's up to the host as to how it is displayed.  Closing as external.
